### PR TITLE
Updates related to operator status and static pod.

### PR DESCRIPTION
**Pull request by aveshagarwal**
@sjenning @sttts

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@sttts @sjenning lgtm?

File: bindata/v3.11.0/kube-scheduler/pod.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: openshift-kube-scheduler
   labels:
     app: openshift-kube-scheduler
+    deployment-id: "DEPLOYMENT_ID"
```
Comment (sttts) [inline bindata/v3.11.0/kube-scheduler/pod.yaml]: we updated this to REVISION, compare @deads2k's recent library-go PR.

Comment (aveshagarwal) [reply bindata/v3.11.0/kube-scheduler/pod.yaml]: @sttts thanks, will update soon.

**Comment by aveshagarwal** [comment]
@sttts I have updated it. Right now I have changed this PR to WIP as there are lot of code changes and i want to make sure that tests pass correctly and scheduler works as expected.

**Comment by aveshagarwal** [comment]
So i noticed that scheduler pods are coming up fine and other pods are getting scheduled. Monitoring if any failures are related to scheduler.

**Comment by aveshagarwal** [comment]
@sttts PTAL

**Comment by sttts** [comment]
Static pod changes lgtm.

**Pull request merged**